Makefile

```diff
@@ -29,17 +29,17 @@ HANDLER_PULL_POLICY ?= Always
 OPERATOR_PULL_POLICY ?= Always
 export IMAGE_BUILDER ?= docker
 
-WHAT ?= ./pkg ./controllers
+WHAT ?= ./pkg/... ./controllers/...
 
-unit_test_args ?=  -r -keepGoing --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_ARGS)
+unit_test_args ?=  -r -keep-going --randomize-all --randomize-suites --race --trace --junit-report=unit-test-junit.xml $(UNIT_TEST_ARGS)
```

Comment:
The new report has little different XSD, as long as you are fine with it, it is ok
you might want to specify the --output-dir $ARTIFACTS ? or the current directory is fine ?

Reply:
Let's see how prow consume it, the output-dir is fine I think, let's check with CI, is the only real user.

Reply:
Let's wait for CI and see how it goes.